boards/arm/atsamc21n_xpro/doc/index.rst

```diff
@@ -0,0 +1,176 @@
+.. _atsamd21_xpro:
+
+SAM C21N Xplained Pro Evaluation Kit
+####################################
+
+Overview
+********
+
+The SAM C21N Xplained Pro evaluation kit is ideal for evaluation and
+prototyping with the SAM C21N Cortex®-M0+ processor-based
+microcontrollers. The kit includes Atmel’s Embedded Debugger (EDBG),
+which provides a full debug interface without the need for additional
+hardware.
+
+.. image:: img/atsamd21_xpro.jpg
```

Comment:
The file is named atsam**d** instead of **c**. Is this a typo?

Reply:
@martinjaeger atmel created two xpro boards: "ATSAMC21N-XPRO" and "ATSAMC21-XPRO". I tested with ATSAMC21N-XPRO

https://www.microchip.com/en-us/products/microcontrollers-and-microprocessors/32-bit-mcus/sam-32-bit-mcus/sam-c

Reply:
Yes I will fix typo :)